app/views/finders/_results.mustache

```diff
@@ -1,3 +1,8 @@
+<div class="feed">
+  <span>Get updates to this list</span>
+  <a href="{{atom_url}}">feed</a>
```

Comment:
I know this is the markup we use in other places (eg Whitehall), but is this OK for screen readers? Should we have more descriptive link text and visually hide it?